save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -94,12 +89,7 @@ class AgentsController(
                     prefix = "save-execution-${execution.id}"
                 )
             }
-                .flatMap { resourcesForExecution ->
```

Comment:
`resourcesForExecution` is not needed here now.
can we move it back to `com.saveourtool.save.orchestrator.service.DockerService#prepareImageAndVolumeForExecution`?

Reply:
Yes, moved